readme.md

```diff
@@ -1,151 +1,132 @@
+- [Mend Renovate Self-Hosted Docs](https://github.com/mend/renovate-ce-ee/tree/main/docs)
 
-> [!NOTE]
-> This extension is created and maintained personally by a Renovate developer/user so support requests relating to the extension itself cannot be answered directly in the main Renovate repository.
+### Comparisons
 
-#### Custom pipeline
+- [Different ways to run Renovate](https://www.mend.io/renovate/)
+- [Renovate vs Dependabot](https://docs.renovatebot.com/bot-comparison/)
 
-You can create a custom pipeline with a `yml` definition that triggers `npx renovate`.
-[More details on how to configure the pipeline](https://docs.renovatebot.com/modules/platform/azure/).
+## Get involved
 
-### Bitbucket Cloud/Server, Forgejo, Gitea, GitLab
+### Issues and Discussions
 
-For Bitbucket Cloud, Bitbucket Server, Forgejo, Gitea and GitLab, use our self-hosting option.
+Please open a Discussion to get help, suggest a new feature, or to report a bug.
+We only want maintainers to open Issues.
 
-## Configuration
+- [GitHub Discussions for Renovate](https://github.com/renovatebot/renovate/discussions)
 
-Go to our [documentation website](https://docs.renovatebot.com/) to learn how to configure Renovate.
-We have a [full list of configuration options](https://docs.renovatebot.com/configuration-options/).
+### Contributing
 
-To get help with your configuration, go to the [discussions tab in the Renovate repository](https://github.com/renovatebot/renovate/discussions) and open a new "config help" discussion post.
+To contribute to Renovate, or run a local copy, please read the contributing guidelines.
 
-## Self-Hosting
+- [Guidelines for Contributing](https://github.com/renovatebot/renovate/blob/main/.github/contributing.md)
+- Items that need contribution: [good first issues](https://github.com/renovatebot/renovate/contribute)
 
-To run your own instance of Renovate you have several options:
+### Contact and Social Media
 
-- Install the `renovate` CLI tool from npmjs, run it on a schedule (e.g. using `cron`)
-- Run the `renovate/renovate:full` Docker Hub image (same content/versions as the CLI tool), run it on a schedule
-- Run the `renovate/renovate:latest` Docker Hub image if you only use package managers that don't need third-party binaries (e.g. JavaScript, Docker, NuGet, pip)
+The Renovate project is proudly supported and actively maintained by Mend.io.
 
-[More details on the self-hosting development](https://github.com/renovatebot/renovate/blob/main/docs/usage/getting-started/running.md).
+- Contact [Mend.io](http://Mend.io) for commercial support questions.
 
-## Contributing
+Follow us on:
 
-If you want to contribute to Renovate or get a local copy running, please read the instructions in [contributing guidelines](.github/contributing.md).
-To get started look at the [list of good first issues](https://github.com/renovatebot/renovate/contribute).
+- Twitter: [x.com/mend_io](http://x.com/mend_io)
```

Comment:
```suggestion
- Twitter: [x.com/mend_io](https://x.com/mend_io)
```

- Links should use HTTPS

Reply:
> Why? If the page has no secure content and does not take input, what's the point in having the overhead of secure encryption?

The Mend Twitter page does not take input. But users may click on the Mend Twitter link, read a tweet, and then _login_ to Twitter, to tweet back.

The catch is that the user may accidentally log in on a hacked page. Here's the sequence of events:

1. User clicks on HTTP link.
1. Hacker reads the HTTP request, intercepts it, and sends user to a hacker-controlled copy of the real page.
1. The user logs in on the _fake_ page.
1. Hacker logs in for the user to the real page.
1. Hacker sends user to the _real_ page.
1. Hacker now has credentials for the user account.
1. User doesn't know they just got hacked, as to them everything seemed normal.
1. Hacker tries credentials on other accounts from the user. Many users re-use the same password on multiple accounts.

When we use HTTPS links, this kind of attack gets harder to do.

## OWASP recommendations

I'll let smarter people explain why HTTPS is good:

### [OWASP, Insecure Transport](https://owasp.org/www-community/vulnerabilities/Insecure_Transport)

> The application configuration should ensure that SSL is used for all access controlled pages.
> 
> If an application uses SSL to guarantee confidential communication with client browsers, the application configuration should make it impossible to view any access controlled page without SSL. However, it is not an uncommon problem that the configuration of the application fails to enforce the use of SSL on pages that contain sensitive data.
> 
> There are three common ways for SSL to be bypassed:
> 
> - A user manually enters the URL and types “HTTP” rather than “HTTPS”.
> - Attackers intentionally send a user to an insecure URL.
> - A programmer erroneously creates a relative link to a page in the application, failing to switch from HTTP to HTTPS. (This is particularly easy to do when the link moves between public and secured areas on a web site.)
>
> **Examples**
> 
> - Login pages are not SSL protected
> - A publicly accessible page contains a relative link to a protected page which forgets to switch to SSL.
> 
> **Related Attacks**
> 
> - Attackers that are trying to steal login credentials, session IDs or other sensitive information
> - Bypassing SSL by entering HTTP instead of HTTPS
> - Sending insecure URLs of protected pages to the victim (e.g. login page) to trick the victim into accessing the privileged pages via HTTP

### [The OWASP Transport Layer Security Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Security_Cheat_Sheet.html#use-tls-for-all-pages)

> TLS should be used for all pages, not just those that are considered sensitive such as the login page. If there are any pages that do not enforce the use of TLS, these could give an attacker an opportunity to sniff sensitive information such as session tokens, or to inject malicious JavaScript into the responses to carry out other attacks against the user.

Reply:
Thanks for the explanation. I didn't know all that. Much appreciated. I will always use https link from now on. 👍

Reply:
You're welcome! :smile: